pf/tests/testdata/updateprogram/Pulumi.yaml

```diff
@@ -17,6 +17,17 @@ resources:
       requiredInputString: |-
         input1
       setOptionals: [ "${r.requiredInputStringCopy}", a, b ]
+
+  randomNumber:
+    type: muxedrandom:RandomInteger
+    properties:
+      min: 2
+      max: 2
+
+  pseudoRandomNumber:
+    type: muxedrandom:RandomHumanNumber
```

Comment:
`pseudoRandomNumber` is a SDKv2 defined resource, while `randomInteger` is a PF defined resource. 

This gives us a basic end-to-end integration test for the muxer.

CC @t0yv0

Reply:
Brilliant. Thank you!